mongo/conf.yaml.example

```diff
@@ -12,6 +12,11 @@ instances:
     #   - optional_tag1
     #   - optional_tag2
 
+    # Whether or not to disable reads from available replicas.
+    # Use this if any replicas are inaccessible to the agent.
+    #
+    # disable_replica_check: false
```

Comment:
Just a nit: Can we have this be an affirmative choice? Like `enable_replica_check`? Having it be an affirmative typically causes less confusion for people and may be more intuitive.